content/en/docs/tasks/administer-cluster/kms-provider.md

```diff
@@ -222,7 +233,7 @@ Then use the functions and data structures in the stub file to develop the serve
   desired buffer to allow config changes to be processed - a minimum `M` of five minutes is recommend).  Note that no
   API server restart is required to perform KEK rotation.
 
-  {{< caution >}}  
+  {{< caution >}}
   Because you don't control the number of writes performed with the DEK, we recommend rotating the KEK at least every 90 days.
```

Comment:
(nit / aside)
```suggestion
  Because you don't control the number of writes performed with the DEK, 
  the Kubernetes project recommends rotating the KEK at least every 90 days.
```